pom.xml

```diff
@@ -1083,10 +1083,8 @@
                             <link>http://lucene.apache.org/core/${version.org.apache.lucene.javadoc}/core/</link>
                             <link>http://lucene.apache.org/core/${version.org.apache.lucene.javadoc}/analyzers-common/</link>
                             <link>http://lucene.apache.org/core/${version.org.apache.lucene.javadoc}/queryparser/</link>
-                            <link>http://jgroups.org/javadoc/</link>
-                            <!-- for javax.batch in the JSR 352 modules -->
+                            <!-- For Java EE APIs, in particular javax.batch in the JSR 352 modules -->
                             <link>https://docs.oracle.com/javaee/${version.javaee.javadoc}/api/</link>
-                            <link>http://docs.jboss.org/jberet/${version.org.jberet.javadoc}/javadoc/jberet-core/</link>
```

Comment:
I think I would have kept it if we have a clear plan to keep the JSR 352 support. Or do you expect this particular jberet part to be dropped (as you keep the EE part).

Reply:
We will restore JSR 352 support at some point.

However, I'd like to point out that what you're suggesting is the exact opposite of what you wanted for everything else in the POM, namely to only copy the necessary configuration to the new POM and migrate the rest later as we need it (if we need it).

Should I understand that, for the modules that we're sure to restore eventually, you'd like us to move, right now, the configuration from the legacy pom to the parent one? And not for the ones we're not sure about?

I can do what you're asking, but I'll have to move (or copy) some properties from the legacy pom to this one, mainly because empty `<link>` elements trigger an NPE during the build (and if you have a look at the next commits, defining the property only in the legacy pom will lead to empty `<link>` elements).

Reply:
Yeah, I can see how it can look as contradictory but at least in my head, it's a little more subtle. I think it's typically an element we could forget to restore. And if we are sure we will need it at some point, I wouldn't have removed it.

Anyway, it's not a big deal, I can live with it :).

Reply:
I added a commit to move the JSR 352 properties up to the parent POM. I don't thing there is much else to move, that we could forget about, so I'll leave it to that.